docs/mcd.md

```diff
@@ -341,3 +341,11 @@ The Connection Metadata file below matches the default--that is, what you get if
 The left side image below shows what the Schema Viewer will look like with this Connection Metadata file. The right side shows what it will look like if you remove the `<schema>` element from the file.
 
 ![alt text]({{ site.baseurl }}/assets/mcd-schema-viewer-1.png "Schema Viewer with Connection Metadata file")&nbsp;![alt text]({{ site.baseurl }}/assets/mcd-schema-viewer-2.png "Schema Viewer after removing schema element from Connection Metadata file")
+
+# Reuired Fields
```

Comment:
```suggestion
# Required Fields
```

Reply:
Also don't see a connection between this header and the new information for Connection Metadata

Reply:
Updated the changes. Pushed old commit.